index.js

```diff
@@ -6,6 +6,10 @@ const helmet = require("helmet");
 const morgan = require("morgan");
 const userRouter = require('./routes/user');
 const authRouter = require('./routes/auth');
+const postRoute = require('./routes/post');
+const commentsRouter = require('./routes/comments');
+const repliesRouter = require('./routes/reply');
```

Comment:
Please try to use a same naming convention
like `postRoute` for others

`commentsRoute`
`repliesRoute`